utils/e2e-tests/tests/try-runtime-check.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -euo pipefail
+
+# Set up command.
+COMMAND="$1"
+
+# Make temporary test directory.
+TEMPDIR="$(mktemp -d)"
+trap 'rm -rf "$TEMPDIR"; pkill -P "$$"' EXIT
+
+# Run the node.
+"$COMMAND" --dev --base-path "$TEMPDIR" &
+
+# Kepp the node running to have around 3 finalized blocks.
+sleep 40
```

Comment:
Do not use sleep as synchronization method in tests. Instead, run a query in an infinite loop with a smaller sleep (and exit when the whatever conditions are met, in the case 3 blocks are finalized), and apply an overall timeout for the operation.

Reply:
Improved